packages/KStepper/KStepper.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <ol class="k-stepper">
+    <KStep
+      v-for="(step) in steps"
+      :key="`k-step-${step.label.replace(' ', '-')}`"
+      :label="step.label"
+      :state="step.state"
+      :max-label-width="maxLabelWidth"
+    />
+  </ol>
+</template>
+
+<script>
+import KStep from './KStep.vue'
+
+export default {
+  name: 'KStepper',
+  components: { KStep },
+  props: {
+    /**
+     * Array of steps to display
+     */
+    steps: {
+      type: Array,
+      required: true,
+      validator: (items) => !items.length || items.some(i => Object.hasOwn(i, 'label'))
```

Comment:
I think the logic here is wrong
```suggestion
      validator: (items) => items.length && items.every(i => Object.hasOwn(i, 'label'))
```